tests/usbredir_test.go

```diff
@@ -1,3 +1,5 @@
+//go:build !s390x
```

Comment:
We use labels in order to filter tests.

Reply:
The Problem in this case is, that the usbredir code itself is not build on s390x, so the whole function test does not compile on s390x when including this, as it is referencing code that does not exist on s390x.

Reply:
Yes, I didn't considered unsupported platforms when writing this.. My expectation is that on s390x the tests are either not run or run and receive `unsupported platform` kind of error somewhere. The latter would need changes in both `virtctl/usbredir` and `tests/usbredir_test.go`... so I'd go with former (this PR) for now.

Reply:
I see, I forgot we import the `virtctl/usbredir` and that is set to not be build... I do wonder why we want to not build the package, @victortoso ? Imho the more straightforward thing is to not include the command. There is nothing in the package that would disallow it to be build on s390x. Then we could remain consistent here and use labels.

Reply:
s390x is big-endian and that's, afaict, is not handled by usbredir protocol.
we can still _build_ on s390x but we should error out that platform is unsupported on runtime.
host-size, qemu's usbredir device driver is not included in [c9s packages](https://mirror.stream.centos.org/9-stream/AppStream/s390x/os/Packages/)

Reply:
Yes, USB is not supported on s390x. The current behavior for kubevirt is to display an error client side in virtctl for usbredir and additionally return an error in the virt-api when trying to use it against an s390x VM.